gamemaster.py

```diff
@@ -37,6 +38,11 @@ def spawn_basic_grunt(self):
         the_basic_grunt = BasicGrunt(starting_position, self.universe_)
         self.universe_.create_enemy(the_basic_grunt)
 
+    def spawn_not_so_basic_grunt(self):
+        starting_position = [self.universe_.width_*random.choice(self.basic_grunt_x_spawn_locations), 0]
```

Comment:
Instead of using `self.basic_grunt_x_spawn_locations` for `not_so_basic_grunt` we should either generalize the spawn location variable name, or make specific one for not so basic grunt. I think the first solution is better.